recipe/meta.yaml

```diff
@@ -13,33 +13,31 @@ source:
     - patches/make_install.patch
 
 build:
-  number: 4
+  number: 5
   skip: true  # [win]
 
 requirements:
   build:
     - {{ compiler('cxx') }}
+    - make
+    # All of this can probably be dropped with GAP 4.11
     - libtool
     - m4
-    - autoconf   # for simpcomp and NormalizInterface
-    - automake   # for NormalizInterface
-    - git        # for NormalizInterface
-    - make
+    - autoconf
+    - automake
+    - git
   host:
     # Since we are using bin/buildpackages.sh to build all packages at once,
     # these requirements are needed.
     - gmp
     - readline {{ readline }}
     - ncurses    # for Browse
     - curl       # for curlInterface
-    - boost-cpp  # for NormalizInterface
     - mpfr       # for float
     - mpfi       # for float
     - mpc        # for float
-    #- fplll     # for float
     - pari       # for alnuth
     - zeromq     # for ZeroMQInterface
-    - linbox     # linboxing
     - libsemigroups   # for Semigroups
```

Comment:
All these dependencies are repeated down below, so I'd assume you'd want to adjust them there, too?

I find it rather crazy that you apparently are forced to duplicate this long list of dependencies. Ouch.

Reply:
Well, different parts of a package have different dependencies. There's a lot of overlap. If they were the same we could alias them. But they are not so well there's not much you can do.